### DISCUSSION TITLE
Crashes when displaying season on one series

### DISCUSSION THREAD

**Discussion by debmint** [jellyfin-archive/jellyfin-roku-legacy#1960] [General]
I don't know if this should be an issue or not, but on one particular Series - Topper (1953) - jellyfin-roku crashes when I display the season list.  This does not happen if viewing on a web browser.  On Roku, if I click on Season 1 (the only season I have), it flashes the Season list momentarily then reverts to the Roku home screen.  I have removed all .nfo and .jpg files and let it rebuild but no improvement.

I am running jellyfin v10.9.11 as a native install.  I also have jellyfin V10.9.10 as a docker container.  They access different media structures and both do the same with this series.

Here is my directory listing for Topper:

.:
 backdrop2.jpg   folder.jpg  'Season 01'   tvshow.nfo

'./Season 01':
S01E12.mp4        S01E19.nfo        S01E27-thumb.jpg  S01E31.mp4
S01E12.nfo        S01E19-thumb.jpg  S01E28.mp4        S01E31.nfo
S01E12-thumb.jpg  S01E20.mp4        S01E28.nfo        S01E31-thumb.jpg
S01E14.mp4        S01E20.nfo        S01E28-thumb.jpg  S01E39.mp4
S01E14.nfo        S01E20-thumb.jpg  S01E29.mp4        S01E39.nfo
S01E14-thumb.jpg  S01E22.mp4        S01E29.nfo        S01E39-thumb.jpg
S01E17.mp4        S01E22.nfo        S01E29-thumb.jpg  season.nfo
S01E17.nfo        S01E22-thumb.jpg  S01E30.mp4
S01E17-thumb.jpg  S01E27.mp4        S01E30.nfo
S01E19.mp4        S01E27.nfo        S01E30-thumb.jpg

At an earlier stage I got an error in the logs like this for each image for this series:
[ERR] Cannot compute blurhash for "/media/streaming/jellyfin/TV_Shows/Topper (1953)/Season 01/Topper (1953) S01E39-thumb.jpg"

I have renamed all my .mp4's to simply be in the format: "SxxExx.mp4", as this is the format for most of my TV shows.

I turned on debugging and then tried to select this season on Roku but nothing relating to this showed up in the logs. Of course, I was accessing the logs from the web browser, as there is no provision AFAIK to do this from roku, but the log is a server log, and anything happening on roku would show up here, would it not?

Could this be a playback issue for Roku?  Or could it be that one or more of the videos has some kind of flaw that is messing up the .nfo file(s)?  I have not tried it yet, but might it help to pass each video through ffmpeg doing a straight copy of both audio and video?

Summing up, I don't think it's a server issue unless the files are messed up in some way that the web interface can handle but not jellyfin-roku.  These files are public domain and downloaded fro archive.org.

Thanks for any help that can be provided.

I did see one pull request that **could** be related. I wasn't able to see any discussion that explained what this was all about, but my problem **seems** to be with only this series, because AFAIK this is the only one that does this.

**Comment by cewert** [comment]
Sounds like a bug to me so I converted this to a bug report. #1961
